packit_service/worker/events/event.py

```diff
@@ -445,10 +445,12 @@ def get_package_config(self) -> Optional[PackageConfig]:
             fail_when_missing=self.fail_when_config_file_missing,
         )
 
+        # TODO do we need to do this?
         # job config change note:
         #   this is used in sync-from-downstream which is buggy - we don't need to change this
-        if package_config:
+        if package_config and self.__class__.__name__ != "NewHotnessUpdateEvent":
```

Comment:
Would it be possible to revert this check (`self.__class__.__name__  == ... `) so we don't need to enhance this when adding any other event?

Reply:
sorry, I am not getting it :sweat_smile: could you please explain how would the change look like?

Reply:
With your change, we are checking if it is not a newly added event. We will need to do this again when adding another event. I was suggesting checking for a case where this is required (so, name the events that require this).

So, instead of `self.__class__.__name__ != Something` (and `self.__class__.__name__ not in {"NewHotnessUpdateEven", "AnotherEventAddedInTheFuture"}` in the future), use `self.__class__.__name__  == "SomeEvent"` or `self.__class__.__name__  in {"SomeEvent", "AnotherEvent"}`

Reply:
Also, could you please use `NewHotnessUpdateEvent.__name__` so type checks are able to realise problems?

Reply:
I wanted to, but that would cause circular import

Reply:
resolved via chat: we will not need to update the condition for each new event in future, this is specific for NewHotnessUpdateEvent, since in its case we will rely on `upstream_project_url` found in the config and do not want to rewrite it